docker-compose.yml

```diff
@@ -23,15 +23,17 @@ services:
       - RASA_URL=${RASA_URL:-http://rasa:5000}
       - DUCKLING_URL=${DUCKLING_URL:-http://duckling:8000}
       - MODEL_DIR=${MODEL_DIR:-./local-storage/rasa/nlu-model}
+      - DOCKER_HOST=${DOCKER_HOST:-localhost}
```

Comment:
What does `DOCKER_HOST` do? and why do we need to tell the API where it is?

Reply:
I had this because it won't always be localhost where docker is if they are on windows for example so its a way to have a IP in there that tells the rest of the stuff where to look since its not localhost